tests/cnf/core/network/sriov/tests/operatorcfg-defaults.go

```
@@ -0,0 +1,182 @@
+const sriovOperatorConfigResourceName = "default"
+
+var sriovOperatorConfigGVR = schema.GroupVersionResource{
+	Group:    "sriovnetwork.openshift.io",
+	Version:  "v1",
+	Resource: "sriovoperatorconfigs",
+}
+
+var _ = Describe("SriovOperatorConfigDefaults", Ordered, Label(tsparams.LabelOperatorConfigDefaultsTestCases),
+	ContinueOnFailure, func() {
+		var (
+			originalInjectorVal      bool
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)